GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -941,6 +940,7 @@ private void InternalInitialize()
                 OnActivate();
 
                 LoadUserMenu();
+                toolStripButtonLevelUp.Image = validBrowseDir && Module.SuperprojectModule is not null ? Images.NavigateUp : Images.SubmodulesManage;
```

Comment:
This adds about 4 ms for me when "spinner" is shown.
It could be moved to later, maybe PostBrowseInitialize (that seem not to be used now).
Too little to care about.

Note: Module.SuperprojectModule is set when GitModule is evaluated when GitModule() is created (using File I/O), so no extra time here.